adam-core/src/test/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDDSuite.scala

```diff
@@ -453,47 +454,57 @@ class AlignmentRecordRDDSuite extends ADAMFunSuite {
     val inputPath = resourcePath("small.sam")
     val reads: AlignmentRecordRDD = sc.loadAlignments(inputPath)
     val outputPath = tmpLocation(".sam")
-    reads.saveAsSam(outputPath, true)
+    reads.saveAsSam(outputPath, asType = Some(SAMFormat.SAM))
```

Comment:
Yeah, it isn't strictly necessary, but I wanted to have the unit tests hit both the case where SAM was identified by extension and where the SAM format was explicitly set.